tfjs-backend-webgpu/src/unary_op_util.ts

```diff
@@ -52,12 +57,31 @@ const ACOS = `
   }
   return acos(a);
 `;
+const ACOSH = `
+  if (a < 1.) {
+    return uniforms.NAN;
+  }
+  return acosh(a);
+`;
 const ASIN = `
   if (abs(a) > 1.) {
     return uniforms.NAN;
   }
   return asin(a);
 `;
+const ASINH = `return asinh(a);`;
+const ATAN = `
+  if (isnan(a)) {
+    return uniforms.NAN;
+  }
+  return atan(a);
+`;
+const ATANH = `
+  if (a >= 1.) {
```

Comment:
`if ((a < -1.0) || (a > 1.0))`?

Reply:
changed to `if (abs(a) >= 1.)`, `a == 1` is also an invalid input data.